recipes/dj-static/meta.yaml

```diff
@@ -0,0 +1,40 @@
+{% set package = "dj-static" %}
```

Comment:
Well, if we do keep the `package` variable, I'd like to vote for calling it `name` so it is more clear.